### DISCUSSION TITLE
Dynamic Maximum Value in Bullet Chart or Gauge Chart

### DISCUSSION THREAD

**Discussion by lauslzrm** [apache/superset#28647] [Ideas]
**Description**
Hello everyone,

I am using Apache Superset for data visualization and would like to implement 'Bullet Chart' or 'Gauge Chart' where the maximum value can be set dynamically based on a metric. Currently, Superset allows setting the maximum value of these charts manually, but I need this value to adjust automatically as the data changes.

**Problem Details**
- Chart Type: Bullet Chart, Gauge Chart
- Requirement: Set the maximum value dynamically based on a metric.
- Current Situation: Maximum value is set manually.
- Need: Both the maximum value and the represented value should be dynamic metrics that update automatically with the data.

**Additional Comments**
Any suggestions or solutions are welcome, as I am currently stuck with this issue.

**Comment by phuochh201** [comment]
Yes, I have the same question as you

**Comment by rithysokun** [comment]
I got the same question too, currently the max value is either entered manually by the user or it generate random max number without using the metric as max value .

**Comment by atamgave1** [comment]
I am also stuck with this issue. anyone has a solution for the same?

**Comment by xavier-GitHub76** [comment]
I also have this need because these elements are actually only static in version 4.0.2.
